### DISCUSSION TITLE
Need 8 core, 16 and 32 core virtual machine in codespace

### DISCUSSION THREAD

**Discussion by Bharani77** [community/community#81732] [Codespaces] [answered]
### GitHub Codespace

Question

Hey! I am unable to see the 8, 16, 32 core virtual machine shapes in my codespace. I really want 32 now for my current work. May i know when i will be getting it?

**Comment by Bharani77** [comment]
Any update on this question??

**Comment by Bharani77** [comment]
No update on this topic?

**Comment by Bharani77** [comment]
No update still?? Then why their is discussion page in github

**Comment by AndnixSH** [comment, marked as answer]
You can make a request to GitHub's official support team by filing a ticket here: https://support.github.com/contact?tags=rr-codespaces%2Ccat_codespaces. That's how I got 8 core, and 16 cores unlocked, except 32 cores (didn't need it anyway)

![image](https://github.com/community/community/assets/40742924/64c05586-6af9-41ca-942c-d4e18c9e2305)

**Reply by Bharani77** [reply]
Thank you so much. I will raise the ticket to get 16 or 32 core..

**Reply by Bharani77** [reply]
Do we need to include or add payment method to enable 8 or 16 core cpu? Can’t we get it without adding payment option?

**Reply by AndnixSH** [reply]
No, you can't get them immediately even you add payment method or buy Github Pro, I tried before. Only way is contact support

**Reply by GinKun43** [reply]
Damn! I have github pro
how long they can respond to me and would i need to pay to use it?
Ty Bro!

**Reply by AndnixSH** [reply]
within 48 hours I think, response was fast for me, and yes you need to pay if you reached quota limit. having Github Pro gives you extra free quotas

**Reply by GinKun43** [reply]
I have github pro
![image](https://github.com/community/community/assets/76711500/445753c3-f4a7-4d85-a8eb-fcac8c2e3783)
but it took almost 2 days since I wrote a ticket to them
![image](https://github.com/community/community/assets/76711500/8a094c46-0fd6-41be-8407-38960f23bf71)
do it charged you any money when you used that? (I mean the codespaces)

**Reply by AndnixSH** [reply]
Yes, I need to pay for it anyway since I use it more than free quotas
![image](https://github.com/community/community/assets/40742924/4779b989-2a51-423b-a95e-60ca272ab5f1)
